corehq/apps/userreports/views.py

```diff
@@ -1482,6 +1483,7 @@ def process_url_params(params, columns):
 @api_auth_with_scope(['reports:view'])
 @require_permission(HqPermissions.view_reports)
 @swallow_programming_errors
+@api_throttle
 def export_data_source(request, domain, config_id):
```

Comment:
@sravfeyn FYI in case this impacts the Solutions analytics work

Reply:
Thanks for flagging! These calls get triggered only on user actions, so I am assuming they aren't fast enough to hit throttling.